cmd/osv-scanner/fixtures/locks-many/osv-scanner.toml

```diff
@@ -0,0 +1,4 @@
+[[IgnoredVulns]]
```

Comment:
Having this here arguably waters the test, since the test cases don't expect an ignore file - is there a reason why you've using recursive on this repo anyway?